vagrant_repo/data_bags/cerner_splunk/alerts-vagrant.json

```diff
@@ -19,7 +19,8 @@
         "reportServerEnabled":false
       }
     },
-    "forwarder":null
+    "forwarder":null,
+    "heavy_forwarder":null
```

Comment:
With the removal of the heavy_forwarder type I think this is also no longer needed.